holidays/countries/timor_leste.py

```diff
@@ -80,7 +80,7 @@ def _populate_public_holidays(self):
             # Veteran's Day.
             self._add_holiday_mar_3(tr("Dia dos Veteranos"))
 
-        # World Labor Day.
+        # International Labor Day.
```

Comment:
```suggestion
        # International Worker's Day.
```
It will be more accurate, and same as in Angola now (also a Portuguese-speaking country).

Reply:
Fixed.